src/__tests__/SelectCategory.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import SelectCategory from "../components/SelectCategory";
+import { render, cleanup, RenderResult } from "@testing-library/react";
+
+afterEach(cleanup);
+
+describe("SelectCategory", () => {
+    const selectCategoryArr = [
+      "HTML",
+      "CSS",
+      "JavaScript",
+      "Accessibility",
+      "General CS",
+      "IT",
+      "Linux",
+      "Python",
+      "SQL"
+    ];
+  
+    it("renders without crashing", () => {
```

Comment:
We should change the title of this test to sometime else like "displays the Choose a Category screen"